sbc_compassion/models/import_letter_line.py

```diff
@@ -40,8 +42,8 @@ class ImportLetterLine(models.Model):
 
     status = fields.Selection([
         ("lang", _("Error in Language")),
-        ("sponsor", _("Error in Sponsorship")),
-        ("encourager", _("Sponsorship not found")),
+        ("sponsorship", _("Sponsorship not Found")),
+        ("person", _("Error in Partner or Child")),
```

Comment:
This error is not clear to me : error in partner or child. What does it mean exactly ?